src/webview/html/messageAsHtml.js

```diff
@@ -118,7 +119,13 @@ $!${divOpenHtml}
 
   const { sender_full_name } = message;
   const sender_id = message.isOutbox ? backgroundData.ownUser.user_id : message.sender_id;
-  const avatarUrl = getAvatarFromMessage(message, backgroundData.auth.realm, 80);
+  const avatarUrl = getAvatarFromMessage(
+    message,
+    backgroundData.auth.realm,
+    // 48 logical pixels; see `.avatar` and `.avatar img` in
+    // src/webview/static/base.css.
+    PixelRatio.getPixelSizeForLayoutSize(48),
```

Comment:
This comment (and the similar one) are quite helpful for understanding this bit of code. But actually even more valuable, and complementary to these, would be a comment pointing in the other direction: from the relevant spot in our CSS to these bits of HTML-generating code.

That's because if someone in the future goes to change the size of these avatars, they'll naturally be looking at the CSS because that's what they have to edit to change it, but they wouldn't necessarily think of this code at all. So a comment there will increase our chances of keeping them in sync.